src/pybind/mgr/dashboard/frontend/src/defaults.scss

```diff
@@ -1,11 +1,46 @@
-$warning-background-color: #fff3cd;
+/*The file suppose to contain all variables, mixins and extends*/
+
+/*Health status*/
+$color-solid-red: #ff0000;
+$color-orange: #ff6600;
+$color-bright-yellow: #ffc200;
+$color-bright-green: #00bb00;
+
+/*Login Form*/
+$color-light-gray: #ececec;
+$color-mild-gray: #555555;
+$color-dark-gray: #474544;
 $oa-color-blue: #288cea;
-$oa-color-light-blue: #afd9ee;
+$color-cornflower-blue: #66afe9;
+
+/*Table Component*/
+$color-off-white: #f5f5f5;
+$color-mild-white: #f6f6f6;
+$border-color-white: 1px solid rgba(0,0,0,.09);
+$color-light-white: #f9f9f9;
+$color-solid-white: #fafafa;
+$color-gallery-gray: #ededed;
+$border-color-gray: 1px solid #d1d1d1;
```

Comment:
I think you should split that and set the border style in the table component scss.

Reply:
Ack